README.md

```diff
@@ -83,12 +83,16 @@ Parameter | Description | Default
 `master.alarms.storageclass` | The storage class for the persistent volume claim of the master's alarm log, mounted to `/var/lib/netdata` | `standard`
 `master.alarms.volumesize` | The storage space for the PVC of the master alarm log | `100Mi`
 `master.env` | Set environment parameters for the master statefulset | `{}`
+`master.podLabels` | Additional labels to add to the master pods | `{}`
```

Comment:
Could you provide an example towards the end of the readme of how to add two labels or annotations? I'd like to make the README as idiot-proof as possible.

Reply:
Done, I added a small section at the end for setting some labels and annotations, lemme know if it looks good